contracts/adapters/FlexibleLeverageStrategyAdapter.sol

```diff
@@ -469,6 +598,43 @@ contract FlexibleLeverageStrategyAdapter is BaseAdapter {
         return _calculateCurrentLeverageRatio(currentLeverageInfo.collateralValue, currentLeverageInfo.borrowValue);
     }
 
+    /**
+     * Calculates the total notional rebalance size. Note: this function does not take into account timestamps, so it may return a nonzero value
+     * even when shouldRebalance would return ShouldRebalance.NONE for all exchanges (since minimum delays have not elapsed)
+     *
+     * @return uint256      total notional rebalance size
+     */
+    function getTotalRebalanceNotional()  external view returns(uint256) {
```

Comment:
For getTotalRebalanceNotional(), think we want to return a bool and an uint. The bool being whether its lever or delever, and the uint is the quantity in whatever asset is being sold (sent to the exchange). So for delever, it is notional quantity in WETH (sold for USDC) and for lever it is notional USDC (sold for WETH). This makes the UX much better for keeper bots. Currently, we’re only returning the notional in collateral asset both directions

For lever, we can convert the collateral notional into borrow units using this internal function _calculateBorrowUnits